com.gitlab.ColinDuquesnoy.MellowPlayer.yml

```diff
@@ -0,0 +1,37 @@
+app-id: com.gitlab.ColinDuquesnoy.MellowPlayer
+runtime: org.kde.Platform
+runtime-version:  "5.15"
+base: io.qt.qtwebengine.BaseApp
+base-version: "5.15"
+sdk: org.kde.Sdk
+command: MellowPlayer
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --device=dri
+  - --socket=session-bus
```

Comment:
Something like this should work for mpris:

```
    "--talk-name=org.mpris.MediaPlayer2.Player",
    "--own-name=org.mpris.MediaPlayer2.MellowPlayer.*"
```

Reply:
Thank you I'll try that

Reply:
That work fine.